publications-2019.bib

```diff
@@ -1,5 +1,37 @@
 % Encoding: US-ASCII
 
+@Article{NW19,
+  author =       {N. Noii and T. Wick},
+  title =        {A phase-field description for pressurized and non-isothermal
+                  propagating fractures},
+  note =      {submitted},
+  year =         {2019},
+  url =       {https://arxiv.org/abs/1904.00196}
+}
+
+@Article{AE19,
+  author =       {J. C. Araujo Cabarcas and C. Engstr{\"o}m},
+  title =        {Removal of spurious solutions encountered in Helmholtz scattering resonance computations in $R^d$},
+  note =      {submitted},
+  year =         {2019},
+  url =       {https://arxiv.org/abs/1904.08812}
+}
+
+
+@article{Mehnert2019,
+  doi = {10.1016/j.euromechsol.2019.103797},
+  url = {https://doi.org/10.1016/j.euromechsol.2019.103797},
+  year = {2019},
+  month = sep,
+  publisher = {Elsevier {BV}},
+  volume = {77},
+  pages = {103797},
```

Comment:
This doesn't look right. Shouldn't there be a hyphen?

Reply:
That's how it's listed on the website by the publisher. I suspect what this is really meant to indicate is an electronic article number, rather than the pages in a printed edition. bibtex doesn't have good ways of expressing this.